docs/installation.rst

```diff
@@ -14,7 +14,14 @@ to other software (such as `msprime <https://github.com/tskit-dev/msprime>`_). I
 as a standalone Python module, users are encouraged to install an official release from PyPI
 using pip::
 
-    $ python -m pip install tskit
+    $ python3 -m pip install tskit
 
-For technical reasons it is not possible to install a development version directly using the
-GitHub URL (i.e. ``pip install git+git://...`` will not work)
+********************
+Development versions
+********************
+
```

Comment:
Need some "we do not recommend this for normal use" statement.

Reply:
Done